src/main/java/io/github/hypixel_api_wrapper/HypixelAPI.java

```diff
@@ -27,15 +28,24 @@ public static HypixelAPI create(String key) {
      * Creates a new instance of the HypixelAPI object
      *
      * @param key             the api key to use for authentication
-     * @param cachingStrategy the caching strategy to use.
+     * @param cachingStrategy the caching strategy to use. If null is passed the NoCachingStrategy
+     *                        will be used to disable caching
      * @return the newly created instance
      */
     public static HypixelAPI create(String key, CachingStrategy cachingStrategy) {
```

Comment:
Dependency injection; not sure if we want to do that now, but in the future we might want to let users supply their own `RequestFactory` here and pass that into the constructor. That way they can use their choice of HTTP client rather than ours, especially if they're already using one for something else.